backend/src/services/aws/s3/s3.service.ts

```diff
@@ -31,6 +33,18 @@ class S3 {
   public async deleteBucket(name: string): Promise<DeleteBucketCommandOutput> {
     return this.#s3Client.send(new DeleteBucketCommand({ Bucket: name }));
   }
+  public async uploadObject({
+    name,
+    file,
+    key,
+  }: {
+    name: string;
+    file: Buffer;
+    key: string;
+  }): Promise<PutObjectCommandOutput> {
+    return this.#s3Client.send(
+      new PutObjectCommand({ Bucket: name, Body: file, Key: key }),
+    );
+  }
```

Comment:
You already have BSObject service. What's the purpose of this method? If you need s3Client in BSObject service just inject it there

Reply:
this method encapsulates logic from s3client.

Reply:
> You already have BSObject service. What's the purpose of this method? If you need s3Client in BSObject service just inject it there

Space Service uses S3 service to work with AWS, Why then BSObject service must do it in a different way?

Reply:
Oh, I missed that it is an aws service 😕